src/escrow/VotingEscrowIncreasing_v1_4_0.sol

```diff
@@ -175,6 +178,15 @@ contract VotingEscrowV1_4_0 is
         emit MinDepositSet(_minDeposit);
     }
 
+    /// @notice Split disabled by default, only whitelisted addresses can split.
+    function setEnableSplit(
+        address _account,
+        bool _isWhitelisted
+    ) external auth(ESCROW_ADMIN_ROLE) {
+        splitWhitelisted[_account] = _isWhitelisted;
```

Comment:
Should have an option to allow any address aka some magic value SPLIT_ANY_ADDRESS